src/subcommand/init.rs

```diff
@@ -36,20 +38,24 @@ impl Init {
         let stdout = io::stdout();
         let mut handle = stdout.lock();
 
-        writeln!(handle, "{}", config.z).unwrap();
+        writeln!(handle, "{}", config.z)?;
         if !self.no_define_aliases {
-            writeln!(handle, "{}", config.alias).unwrap();
+            writeln!(handle, "{}", config.alias)?;
         }
 
         match self.hook {
             Hook::none => (),
-            Hook::prompt => writeln!(handle, "{}", config.hook.prompt).unwrap(),
+            Hook::prompt => writeln!(handle, "{}", config.hook.prompt)?,
             Hook::pwd => {
                 if let Some(pwd_hook) = config.hook.pwd {
```

Comment:
I'd prefer a `match` clause over here - they tend to be more readable when you want to support both cases.